vnext/src/RNTester/AccessibilityExample.tsx

```diff
@@ -4,21 +4,80 @@
 
 class AccessibilityBaseExample extends React.Component {
   public render() {
     return (
       <View>
         <Text>The following has accessibilityLabel and accessibilityHint:</Text>
         <View
-          style={{width:100, height:50, backgroundColor:'blue'}}
+          style={{width:50, height:50, backgroundColor:'blue'}}
           accessibilityLabel="A blue box"
           accessibilityHint="A hint for the blue box."
         />
         <Text>The following has accessible and accessibilityLabel:</Text>
         <View
-          style={{ width: 100, height: 50, backgroundColor: 'red' }}
+          style={{ width: 50, height: 50, backgroundColor: 'red' }}
+          accessible={true}
+          accessibilityLabel="A hint for the red box."
+        />
+      </View>
+    );
+  }
+}
+
+class HighContrastExample extends React.Component {
+  state = {
+    isHighContrast: AppTheme.isHighContrast,
+    highContrastColorValues: AppTheme.currentHighContrastColorValues,
+    currentTheme: AppTheme.currentTheme
+  };
+
+  componentDidMount() {
+    AppTheme.addListener('highContrastChanged', this.onHighContrastChanged);
+    AppTheme.addListener('appThemeChanged', this.onAppThemeChanged);
+  }
+
+  componenetWillUnmount() {
+    AppTheme.removeListener('highContrastChanged', this.onHighContrastChanged);
+    AppTheme.removeListener('appThemeChanged', this.onAppThemeChanged);
+  } 
+
+  // TODO: Make args props
+  onHighContrastChanged = (event: IAppThemeChangedEvent) => {
+    this.setState({isHighContrast : AppTheme.isHighContrast, 
+                  highContrastColorValues : AppTheme.currentHighContrastColorValues});
+  };
+
+  onAppThemeChanged = (event: any) => {
+    this.setState({currentTheme : AppTheme.currentTheme});
+  }
+
+  public render() {
+    return (
+      <View>
+        <Text>The following has HighContrast Event awareness:</Text>
+        <View
+          style={{width: 150, height: 50, backgroundColor: currentBackgroundColor[2]}}
+          accessibilityLabel="A blue box"
+          accessibilityHint="A hint for the blue box.">
+        <Text>isHighContrast: {this.state.isHighContrast ? 'true' : 'false'}</Text>
+        </View>
+        <Text>ButtonFaceColor value: {this.state.highContrastColorValues.ButtonFaceColor}</Text>
+        <Text>ButtonTextColor value: {this.state.highContrastColorValues.ButtonTextColor}</Text>
+        <Text>GrayTextColor value: {this.state.highContrastColorValues.GrayTextColor}</Text>
+        <Text>HighlightColor value: {this.state.highContrastColorValues.HighlightColor}</Text>
+        <Text>HighlightTextColor value: {this.state.highContrastColorValues.HighlightTextColor}</Text>
+        <Text>HotlightColor value: {this.state.highContrastColorValues.HotlightColor}</Text>
+        <Text>WindowColor value: {this.state.highContrastColorValues.WindowColor}</Text>
+        <Text>WindowTextColor value: {this.state.highContrastColorValues.WindowTextColor}</Text>
```

Comment:
nit: A more interesting example would be doing a simple version of what's shown in PC settings. If highcontrast is disabled, then grey out the view. If it is enabled, then show the colors in action on buttons/text.